apps/src/sites/studio/pages/init/loadGamelab.js

```diff
@@ -8,6 +8,10 @@ export default function loadGamelab(options) {
   options.skinsModule = skins;
   var gamelab = new GameLab();
 
+  if (IN_UNIT_TEST) {
+    window.Gamelab = gamelab;
+  }
```

Comment:
I actually prefer that we don't just put Gamelab on window - I think it pressures us to design certain things "right."

For what it's worth, Caley and I also discussed having this method return the Gamelab instance, which the test entry point could put on window if it wanted to.  That would essentially remove any test-specific code from this file, and not expose Gamelab on window at all in production code.